src/backend/joanie/payment/backends/lyra/__init__.py

```diff
@@ -319,6 +319,35 @@ def create_zero_click_payment(self, order, installment, credit_card_token):
 
         return True
 
+    def is_already_debited(self, order, installment):
```

Comment:
IMO the `BasePaymentBackend` should implement this method which would just raise a `NotImplemented` exception